src/middlewares/signupDataValidate.js

```diff
@@ -0,0 +1,24 @@
+const emailValidator = require("email-validator");
+const signupDataValidate = (req, res, next) => {
+  const { firstname, lastname, email, password } = req.body;
+  //Verify if every field is fullled
+
+  if (!firstname || !lastname || !email || !password) {
+    return res.status(400).json({
+      success: false,
+      message: "Every field is required",
```

Comment:
Good idea to add some validation! The validation message would be more useful to the client if the message explicitly listed which fields are required. Something like this: `The following fields are required: firstname, lastname, email, password`.

Reply:
Nice Idea . I will add it !